game-core/src/main/java/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -1545,6 +1550,23 @@ public void keyPressed(final KeyEvent e) {
     };
   }
 
+  private String getUnitInfo() {
+    String unitInfo = "";
+    if (unitsBeingMousedOver != null && !unitsBeingMousedOver.isEmpty()) {
+      final Unit unit = unitsBeingMousedOver.get(0);
+      final UnitAttachment ua = UnitAttachment.get(unit.getType());
+      if (ua != null) {
+        String unitText = "Unit:";
+        if (unitsBeingMousedOver.size() != 1) {
+          unitText = unitsBeingMousedOver.size() + " Units";
+        }
```

Comment:
The unitText assignment could be done with a ternary operator, so this variable could be declared final

Reply:
Done, though I find that a bit harder to read.